### PR TITLE
Fixed line break in vote presets on mobile

### PR DESCRIPTION
**Pull request by mktcode**
At around 430px width the 100% button jumps into a new line.

bildschirmfoto vom 2017-10-29 um 23 19 23

I adjusted paddings for everything below 768px, down to 320px.

bildschirmfoto vom 2017-10-29 um 23 20 14

(his smile somehow makes me feel uncomfortable... :D)

### CODE REVIEW

**Comment by bonustrack** [comment]
Thank you for your contribution, this was annoying me :). I just find that a some point i can get it on 2 lines (with window width of 771px):
image

**Comment by mktcode** [comment]
I can't confirm this @bonustrack. Which browser? I tested in Firefox 56 and Chromium 62.

**Comment by mktcode** [comment]
Here is my fifth contributor report. This time I made another few changes in the new busy design. As my changes are quite small most of the time, I don't feel good about posting a report for every single pull request I make and so I will summarize multiple PRs in one report.

## Style Fixes

<hr>

### [Removed border-radius and left/right border from posts on mobile](https://github.com/busyorg/busy/pull/885)
###

The posts on busy.org have a small border-radius and a light grey border. On small screens the posts expand to the full screen width. Removing the border-radius and left and right borders saves us two whole pixels (!) of space and just looks better.



<hr>

### [Fixed .StoryLoading border/border-radius on small screens](https://github.com/busyorg/busy/pull/894)
###

The post placeholder that's visible when a feed is loading had the same issue.



<hr>

### [Padding fixes](https://github.com/busyorg/busy/pull/886)
###

On medium screens there was no space between the content and the browser's boundaries.

https://user-images.githubusercontent.com/6792578/31865829-7ce86f56-b775-11e7-997a-8d95b1abd95c.png

https://user-images.githubusercontent.com/6792578/31865900-fbbf3926-b776-11e7-831a-57ebe33d7ff0.png

<hr>

### [Fixed line break in vote presets on mobile](https://github.com/busyorg/busy/pull/944)
###

I noticed an issue on mobiles with the new slider for the voting power that we have thanks to @sekhmet! I reduced the button's minimum size so it fits even the smallest screens.



## Voting Power

<hr>

### [Add current voting power to UserInfo](https://github.com/busyorg/busy/pull/927)
###

You can now see your own or someone else's current voting power in the left sidebar when viewing the profile page.

<center>https://user-images.githubusercontent.com/6792578/32139370-95d5ce3e-bc46-11e7-9fd8-9b4b2c3968b1.png</center>

**One note on that one:** Thie displayed voting power is **not** very accurate because it's the voting power at the time of the last vote. Most projects I know, including steemstats and steemdb, show this inaccurate value. I would need to calculate the replenished power myself and add it. I will implement this once I am a little bit more familiar with React.js or someone else needs to do it. I provided a code example in the PR.

<hr>

### That's it!
###

This were all the PRs I submitted since the last busy-related contributor report. Small and barely noticeable changes but many of such small changes here and there will make a big difference in the long term. Also I like to be pedantic enough to address all these small annoying optical issues that otherwise would probably be ignored for a long time.

<center>https://images.duckduckgo.com/iu/?u=http%3A%2F%2F2.bp.blogspot.com%2F-41v6n3Vaf5s%2FUeRN_XJ0keI%2FAAAAAAAAN2Y%2FYxIHhddGiaw%2Fs1600%2Fcss.gif&f=1</center>

# <center>See you in the next CR!<br>There's so much to do! :)</center>

<hr>

<center>[](https://steemit.com/@mkt)</center>
<center>@mkt<br><sup>*Interests:*</sup><br>Technology/Science/Future<br>Nature/Environment/Sustainability<br>Creativity/Programming</center>

<hr>

### <center><sub>Check this out:</sub><br>[Steemit On Your Website](https://mktcode.github.io/steemit-widgets/)</center><br /><hr/><em>Open Source Contribution posted via <a href="https://utopian.io/utopian-io/@mkt/busy-org-ui-and-voting-power-contributor-report-5">Utopian.io</a></em><hr/>

**Comment by bonustrack** [comment]
@mktcode i'm using Chrome Version 61.0.3163.100, can you slide the width and spot the moment where it goes into 2 lines? See:
votingslider

**Comment by mktcode** [comment]
@bonustrack I really tried but I can't reproduce it. Are you on windows? Maybe it has something to do with how the font gets rendered. I could just further reduce the minimum padding but I can't check if it's fixed then. But  it only affects this tiny range anyway. At least an improvement. You can check if it's now working for you. Let me quickly change the file...

**Comment by bonustrack** [comment]
I'm on Mac @mktcode. @jm90m are you able to see the button in 2 lines with a browser width of about 771px?

**Comment by jm90m** [comment]
can you deploy this @bonustrack I don't see the link?

**Comment by jm90m** [comment]
screen shot 2017-11-01 at 5 31 10 pm
screen shot 2017-11-01 at 5 30 58 pm

@bonustrack looks fine for me with browser width @ 771px

**Comment by bonustrack** [comment]
I'm not able to reproduce it when i use the console responsive tool. Can you test just by resizing your browser window? @jm90m

**Comment by jm90m** [comment]
Yeah I resized my browser window, it goes to next line @ about under 450px;

screen shot 2017-11-01 at 5 43 29 pm

**Comment by mktcode** [comment]
@jm90m @bonustrack This is indeed weird. Resizing the actual window reveals the issue, while using dev tools everything is fine. Strange... However, i fixed it by simply removing the breakpoint. It defaults now to `padding: 0 3px` on all screen sizes which is ok because the text is centered anyway.

File: src/components/Slider/RawSlider.less [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import (reference) "../../styles/modules/variables";
```
Comment (bonustrack) [inline src/components/Slider/RawSlider.less]: Is this line been used?

Comment (mktcode) [reply src/components/Slider/RawSlider.less]: good catch... thanks :) fixed it

**Pull request merged**